### PR TITLE
feat: add support for additional env vars to the controller deployment

### PR DESCRIPTION
**Pull request by charlie-haley**

### CODE REVIEW

**Review by mloiseleur — APPROVED** [review]
LGTM

File: hub-agent/values.yaml [hunk 1]
```diff
@@ -24,6 +24,9 @@ controllerDeployment:
     metricsURL: http://traefik-hub.hub-agent.svc.cluster.local:9100/metrics
   args:
     - --log-level=debug
+  # env:
+  #   - name: "LOG_LEVEL"
+  #     value: "debug"
```
Comment (youkoulayley) [inline hub-agent/values.yaml:29]: I think we can remove that from here no ?

**Review by youkoulayley — CHANGES_REQUESTED** [review]
LGTM, just minor comment

File: hub-agent/values.yaml (hunk 1 above)
Comment (charlie-haley) [reply hub-agent/values.yaml:29]: I left it there to document the available properties, similar to this https://github.com/traefik/hub-helm-chart/blob/452cffb6fb875f4163942d1d6f54dfb58ddc381f/hub-agent/values.yaml#L66-L71

Comment (youkoulayley) [reply hub-agent/values.yaml:29]: It's a little weird for the log level because it's also defined in the args just above

**Review by youkoulayley — APPROVED** [review]

**Pull request merged**